builtin/logical/pki/path_fetch_issuers.go

```diff
@@ -321,3 +321,89 @@ Writing to /issuer/:ref allows updating of the name field associated with
 the certificate.
 `
 )
+
+func pathGetIssuerCRL(b *backend) *framework.Path {
+    pattern := "issuer/" + framework.GenericNameRegex(issuerRefParam) + "/crl(/pem)?"
+    return buildPathGetIssuerCRL(b, pattern)
+}
+
+func buildPathGetIssuerCRL(b *backend, pattern string) *framework.Path {
+    fields := map[string]*framework.FieldSchema{}
+    fields = addIssuerRefNameFields(fields)
+
+    return &framework.Path{
+        // Returns raw values.
```

Comment:
Is there value in perhaps adding a JSON equivalent (like `/certs/crl`) here too, for anyone using the CLI?

Reply:
Let's do this in another PR.